locales-en-US.xml

```diff
@@ -169,6 +245,50 @@
     </term>
 
     <!-- SHORT LOCATOR FORMS -->
+    <term name="act">			 
+      <single>a.</single> <!-- according to acronyms.com, but is this really common?-->
+      <multiple></multiple>						 
```

Comment:
```suggestion
      <single>act</single>
      <multiple>acts</multiple>						 
```

Indigobook and SBL don't have an abbreviation for act, so let's go without unless there is some complaint

Reply:
Ok. Should we just delete the short term form? (Will it fall back to the long form in the absence of a short form?)

Reply:
Or should we short form entries that repeat the long form?